soh/soh/Enhancements/randomizer/entrance.cpp

```diff
@@ -10,31 +10,23 @@
 namespace Rando {
 EntranceLinkInfo NO_RETURN_ENTRANCE = { EntranceType::None, RR_NONE, RR_NONE, -1 };
 
-Entrance::Entrance(RandomizerRegion connectedRegion_, std::vector<ConditionFn> conditions_met_, bool spreadsAreasWithPriority_)
-    : connectedRegion(connectedRegion_),  spreadsAreasWithPriority(spreadsAreasWithPriority_){
+Entrance::Entrance(RandomizerRegion connectedRegion_, ConditionFn condition_function_, bool spreadsAreasWithPriority_)
+    : connectedRegion(connectedRegion_), condition_function(condition_function_),  spreadsAreasWithPriority(spreadsAreasWithPriority_){
```

Comment:
Would've honestly prefered if the file structure stuff would be completely separate from changes like these because it's super hard to review actual changes when the files are entirely removed and re-added. At this point I basically just have to trust it if I don't want to review every single file separately.